primaryHandler.go

```diff
@@ -213,3 +224,202 @@ func authenticationHandler(v *viper.Viper, logger log.Logger, registry xmetrics.
 	chain := alice.New(constructors...)
 	return &chain, nil
 }
+
+type webhookHandlerIn struct {
+	fx.In
+	V                   viper.Viper
+	WebhookConfigKey    ancla.Config
+	ArgusClientConfigIn ArgusClientTimeoutConfigIn
```

Comment:
We already call `newArgusClientTimeout()` from `fx.Provide()`.  Is this the struct we want here? Should we do this instead?
```suggestion
	ArgusClientConfigIn httpClientTimeout `name:"argus_client_config`
```